src/js/tests.rs

```diff
@@ -0,0 +1,751 @@
+use super::*;
+use crate::build::{Origin, Module};
+use crate::type_;
+
+macro_rules! assert_js {
+    ($src:expr, $erl:expr $(,)?) => {
+        // println!("\n\n\n{}\n", $src);
+        let (mut ast, _) = crate::parse::parse_module($src).expect("syntax error");
+        ast.name = vec!["the_app".to_string()];
+        let mut modules = std::collections::HashMap::new();
+        // Probably a nicer way to do types values accessors
+        modules.insert(
+            "rocket_ship".to_string(), 
+            (Origin::Src, type_::Module { name: vec!["rocket_ship".to_string()], types: std::collections::HashMap::new(), values: std::collections::HashMap::new(), accessors: std::collections::HashMap::new()})
```

Comment:
@tranzystorek-io do you have any clever idea for adding module content here.

I use the module rocket_ship below to test imports but I can't yet test, the following. The module has no functions and building it by hand looks a pain.
```
import rocket_ship.{launch}
```

Reply:
If it's only about making a hashmap literal, then the [maplit](https://docs.rs/maplit/1.0.2/maplit/) crate is the solution

Reply:
Thanks, I had a look at that. In the end I co-opted the parse/compile machinery and generate a dummy module from a string.